src/other/utils.jl

```diff
@@ -178,3 +178,133 @@ function _nt_like_hash(v, h::UInt)
 
     return xor(objectid(Tuple(propertynames(v))), h)
 end
+
+_findall(B) = findall(B)
+
+@inline _blsr(x)= x & (x-1) 
```

Comment:
Also please make sure you do not leave trailing whitespace in other places
```suggestion
@inline _blsr(x)= x & (x-1)

```